### PR TITLE
chore: deprecate Relay

### PR DESCRIPTION
**Pull request by cbachmeier**
- deprecate Relay packages
- remove unnecessary build steps
- remove ignore patterns for removed relay files

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -9,9 +9,6 @@
 /src/locales/**/pseudo.po
 
 # generated graphql types
-# TODO: after apollo migration is finished, simplify gitignore
-/src/**/__generated__/*
```
Comment (zzmp) [inline .gitignore:13]: Are there no other generated files that need to be ignored?

File: package.json [hunk 2]
```diff
@@ -8,18 +8,15 @@
     "contracts:compile:abi": "typechain --target ethers-v5 --out-dir src/abis/types \"./src/abis/**/*.json\"",
     "contracts:compile:v3": "typechain --target ethers-v5 --out-dir src/types/v3 \"./node_modules/@uniswap/**/artifacts/contracts/**/*[!dbg].json\"",
     "contracts:compile": "yarn contracts:compile:abi && yarn contracts:compile:v3",
-    "relay": "relay-compiler relay.config.js",
-    "relay-thegraph": "relay-compiler relay_thegraph.config.js",
     "graphql:fetch": "node fetch-schema.js",
-    "graphql:generate": "yarn relay && yarn relay-thegraph",
     "apollo:generate:data": "graphql-codegen --config apollo-codegen.ts",
```
Comment (zzmp) [inline package.json:12]: These are really graphql-centric, not apollo-centric.
I'd prefer if these scripts could be renamed (to `graphql:generate`, `graphql:generate:data`, `graphql:generate:thegraph`), and the config files could be renamed from `apollo[,_thegraph].config.js` to `graphql[,_thegraph].config.js`.

**Review by zzmp — CHANGES_REQUESTED** [review]
removals all look good. just requesting a rename from apollo to graphql for generic graphql infra.

**Review by zzmp — APPROVED** [review]
once you change from apollo->graphql-centric naming, this lgtm.
i don't need to re-review, so I'm pre-emptively granting approval.

File: .gitignore (hunk 1 above)
Comment (cbachmeier) [reply .gitignore:13]: Nope, we only generate the types-and-hooks files and the schema file

**Pull request merged**